app.polychromatic.controller.json

```diff
@@ -0,0 +1,184 @@
+{
+    "app-id": "app.polychromatic.controller",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "command": "polychromatic-controller",
+    "base": "com.riverbankcomputing.PyQt.BaseApp",
+    "base-version": "6.7",
+    "rename-icon": "polychromatic",
+    "rename-desktop-file": "polychromatic.desktop",
+    "cleanup-commands": [
+        "/app/cleanup-BaseApp.sh"
+    ],
+    "finish-args": [
+        "--env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--share=network",
+        "--talk-name=org.razer",
+        "--device=dri"
+    ],
+    "modules": [
+        "python3-modules.json",
+        {
+            "name": "sassc",
+            "cleanup": ["*"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/sass/sassc/archive/3.6.2.tar.gz",
+                    "sha256": "608dc9002b45a91d11ed59e352469ecc05e4f58fc1259fc9a9f5b8f0f8348a03"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "autogen.sh",
+                    "commands": ["autoreconf -si"]
+                }
+            ],
+            "modules": [
+                {
+                    "name": "libsass",
+                    "cleanup": ["*"],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://github.com/sass/libsass/archive/3.6.6.tar.gz",
+                            "sha256": "11f0bb3709a4f20285507419d7618f3877a425c0131ea8df40fe6196129df15d"
+                        },
+                        {
+                            "type": "script",
+                            "dest-filename": "autogen.sh",
+                            "commands": ["autoreconf -si"]
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "openrazer-pylib",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make PREFIX=/app daemon_install python_library_install"
+            ],
```

Comment:
use `no-autogen: true`, and `make-args`, drop the build-commands.

Reply:
Tried that:
```diff
-   "build-commands": [
-      "make PREFIX=/app daemon_install python_library_install"
-   ],
+   "no-autogen": true,
+   "make-args": [
+      "PREFIX=/app",
+      "daemon_install",
+      "python_library_install"
+   ],
```

But this wouldn't work:

    Error: module openrazer-pylib: Buildsystem simple requires specifying "build-commands"

Unless I've misunderstood your requested change? Or it's because of how the [Makefile](https://github.com/openrazer/openrazer/blob/master/Makefile) is put together?

None of the other build systems would be possible:

> Build system to use: autotools, cmake, cmake-ninja, meson, simple, qmake

Reply:
No like this:

```
            "name": "openrazer-pylib",
            "no-autogen": true,
            "make-install-args": [
                "PREFIX=/app",
                "daemon_install",
                "python_library_install"
            ],
            "sources": [
                {
                    "type": "archive",
                    "url": "https://github.com/openrazer/openrazer/releases/download/v3.9.0/openrazer-3.9.0.tar.xz",
                    "sha256": "273ae58a54645e22ddaf65a627d3fac370328e471e095b735dfa3f0021c7cbeb"
                }
            ]
```

I haven't checked your Makefile targets but you might need to swap between make-args and make-install-args.

Reply:
My bad. Just needed to drop the `buildsystem` key too. Got it!